examples/repl/cmdline.rs

```diff
@@ -914,6 +923,20 @@ pub async fn cmdline(context: Context, line: &str, chat_id: &mut ChatId) -> Resu
                 )
                 .await?;
         }
+        "protect" | "unprotect" => {
+            ensure!(!arg1.is_empty(), "Argument <chat-id> missing.");
+            let chat_id = ChatId::new(arg1.parse()?);
+            chat_id
+                .set_protection(
+                    &context,
+                    match arg0 {
+                        "protect" => ProtectionStatus::Protected,
+                        "unprotect" => ProtectionStatus::Unprotected,
+                        _ => panic!("Unexpected command (This should never happen)"),
```

Comment:
[`unreachable!()`](https://doc.rust-lang.org/std/macro.unreachable.html) is better in this case

Reply:
thanks for the hint, i did a commit :)